.github/workflows/security.yaml

```diff
@@ -0,0 +1,27 @@
+name: Security
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "0 13 * * 1"  # 8am CT on Mondays
+
+jobs:
+  scan-for-vulns:
+    runs-on: ubuntu-latest
+    container:
+      image: public.ecr.aws/opslevel/platform-tools:latest
+      env:
+        ORG_GITHUB_TOKEN: ${{ secrets.ORG_GITHUB_TOKEN }}
+        GRYPE_INTEGRATION_SECRET: ${{ secrets.GRYPE_INTEGRATION_SECRET }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
```

Comment:
Non-blocker, but it looks like `fetch-depth` and `submodules` can be dropped from here and would speed this up a bit